virttest/utils_v2v.py

```diff
@@ -821,6 +825,28 @@ def delete_windows_file(self):
         cmd = "DEL C:\rss.reg.bak"
         return self.run_cmd(cmd)[0]
 
+    def is_uefi_guest(self):
+        """
+        Check whether windows guest is uefi guest
+
+        More info:
+        https://www.tenforums.com/tutorials/85195-check-if-windows-10-using-uefi-legacy-bios.html
+        """
+        search_str = "Detected boot environment"
+        target_file = "c:\Windows\Panther\setupact.log"
+        cmd = 'findstr /c:"%s" %s' % (search_str, target_file)
+        status, output = self.run_cmd(cmd)
+        if status != 0:
```

Comment:
finally ,it will return false, so the code here is duplicated

Reply:
Done